apps/ejabberd/src/ejabberd_auth_jwt.erl

```diff
@@ -57,7 +57,26 @@
 %%%----------------------------------------------------------------------
 
 -spec start(Host :: ejabberd:server()) -> ok.
-start(_Host) ->
+start(Host) ->
+    UsernameKey = get_auth_opt(Host, jwt_username_key),
+    true = is_atom(UsernameKey) andalso UsernameKey /= undefined,
+
+    JWTSecret =
+    case {get_auth_opt(Host, jwt_secret_source), get_auth_opt(Host, jwt_secret)} of
+        {undefined, JWTSecret0} when is_list(JWTSecret0) ->
+            list_to_binary(JWTSecret0);
+        {undefined, JWTSecret0} when is_binary(JWTSecret0) ->
+            JWTSecret0;
+        {undefined, {env, _} = Env} ->
```

Comment:
Do we need this if we allow and say in the doc that `{env, "asdf"}` can be set via `jwt_secret_source`?